playbooks/roles/ocp-cost-management/tasks/main.yml

```diff
@@ -0,0 +1,118 @@
+---
+# tasks file for playbooks/roles/ocp-cost-management
+
+- name: Check if cluster operators and nodes are healthy
+  include_role:
+    name: check-cluster-health
+
+- name: Clean up
+  include_tasks: "{{ role_path }}/tasks/cost_management-cleanup.yml"
+  when: cost_management_cleanup
```

Comment:
I don't see any relevance of "cost_management_cleanup" variable here can u please explain why is it used